### ISSUE TITLE
Address eslint errors in Backend

### ISSUE THREAD

**Issue by jasonwong26** [hackforla/VRMS#647]
# v0.4 Setup Infrastructure

This ticket prepares us to implement the PR Validation actions by ensuring we have a clean environment - with passing test and linting rules.

Note: changes here are temporary.  Once the #582 PR is merged into development we will use those instead.

# Dependencies
- #645

# Instructions
- [ ] Run the `npm run lint` on the backend folder.
- [ ] Address each reported error by either:
  - modifying the code
  - modifying the relevant eslint rule to treat the issue as a warning

# Acceptance Criteria
Running the `npm run lint` command should not return any errors.

**Comment by peteplass** [comment]
Jason turned off es-lint rules so that merges will go through. 

Closing
